hbw/app/services/hbw/task_notifier.rb

```diff
@@ -21,7 +21,13 @@ def call(widget, task_id, event_name, users = [])
           message[:cache_key] = cache_key
         end
 
+        need_to_check_assignee = event_name == 'assignment'
+
         users.each do |user|
+          if need_to_check_assignee
+            message[:is_assigned] = user == assignee
```

Comment:
`is_assigned` is bad name. 
'is' is redundant and there is nothing about assignment to current user.